lib/connectors/mongodb/connector.rb

```diff
@@ -0,0 +1,76 @@
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License;
+# you may not use this file except in compliance with the Elastic License.
+#
+
+# frozen_string_literal: true
+
+require 'mongo'
+require 'connectors/base/connector'
+
+module Connectors
+  module MongoDB
+    class Connector < Connectors::Base::Connector
```

Comment:
I feel pretty weird about a whole connector being added without any tests.

Reply:
I'm gonna add more tests a bit later, as a unified effort - now things get moved around too often to have a stable "framework" definition and distribution of responsibilities.